index.html

```diff
@@ -18,15 +18,16 @@
 	</script>
 	<header id="header">
 		<section class="arabic">
-			<h1>نهاية بلديهما الساحلية</h1>
-			<p>لم وقد وعلى الأهداف الانجليزية, ان انذار والكساد عدم. أضف في اوروبا المشترك, أخذ جيوب مكّن الواقعة عل. تصفح أحكم الاندونيسية وقد ٣٠, كل بين معزّزة محاولات اليابان،, قبل ثم حاول الشتاء الربيع،. لإعادة الشتاء الأعمال في أسر, أم مئات أساسي العالمي نفس. يكن مسرح العالمي وحلفاؤها أم, حيث الله معاملة وانتهاءً قد.</p>
-			<p>كلّ خطّة مسرح والنرويج تم, تم مسرح لغات كثيرة شيء, في مئات الجنوبي ذات. هو جسيمة بتخصيص المنتصر فعل, من بعد وشعار ضمنها انتهت, قبل الآلاف الأولى باستخدام بـ. هذه الله يتمكن بولندا، بل. غرّة، استمرار ثم بها. تم وإيطالي استرجاع يتم, جدول استبدال نفس لم, مرمى تعداد السادس ان ومن. بين مشارف الإمتعاض ما, خطّة مايو وتزويده ٣٠ يبق, كانت المدن واستمرت مدن ان.</p>
+			<h1>فسيفساء سورية</h1>
+			<div id="rectangle"></div>
+			<p>ويهدف التصور من قاعدة البيانات من الأرشيف التراث السورية لضمان عدم نسيان هذا التراث الثقافي السوري. بفضل رقمنة قواعد البيانات، فإنه قد يكون بمثابة أساس لأية إعادة الإعمار المحتملين.</p>
 		</section>
 		<section>
-			<h1>Lorem ipsum dolor</h1>
-			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus id, doloremque libero labore? Voluptatum nostrum quibusdam deserunt ratione a et maiores, nobis eligendi possimus excepturi suscipit aperiam perferendis earum ut.</p>
-			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At quis commodi alias nulla deserunt quo vel harum, odio suscipit enim error ducimus asperiores ratione maxime laborum fugit iusto praesentium quas.</p>
+			<h1>Syrian Mosaic</h1>
+			<div id="rectangle"></div>
```

Comment:
`id`s shoudn't be used twice (see line 22). instead, this can be done using css:

```css
#header section h1 + p::before {
    display: block;
    width: 70%;
    height: .8rem;
    background-color: #34FC8A;
}
```